x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+      "key": "windows.advanced.elasticsearch.tls.verify_peer",
```

Comment:
@ferullo here's where we will add the advanced fields that we'll generate in the form.  Do you have a ballpark on the amount of fields we'll have?  Or better yet, a complete list so far?

Reply:
Here is a complete list so far (sorry to link to a private repo).  https://github.com/elastic/endpoint-dev/blob/master/Documentation/ElasticEndpoint/example_unofficial_config.yaml

Reply:
updating this file to reflect the current list, with no `last_supported_version` keys, though we support them in the front-end code. leaving `documentation` key empty for now. example:
```
  {
      "key": "linux.advanced.agent.connection_delay",
      "first_supported_version": "7.11",
      "documentation": ""
  },
```

Reply:
23238e9d29504f34a9e774cad99443b830951c52